app/metrics.yaml

```diff
@@ -1236,6 +1236,72 @@ private_browsing_mode:
     notification_emails:
       - fenix-core@mozilla.com
     expires: "2020-03-01"
+  settings_shortcut:
+    type: event
+    description: >
+      A user pressed the "Add private browsing shortcut" button in settings.
+    bugs:
+      - 4658
+    data_reviews:
+      - https://github.com/mozilla-mobile/fenix/pull/5194
+    notification_emails:
+      - fenix-core@mozilla.com
+    expires: "2020-03-01"
+  cfr_add_shortcut:
+    type: event
+    description: >
+      A user pressed the "Add shortcut" button when the contextual feature recommender appeared.
+    bugs:
+      - 4658
+    data_reviews:
+      - https://github.com/mozilla-mobile/fenix/pull/5194
+    notification_emails:
+      - fenix-core@mozilla.com
+    expires: "2020-03-01"
+  cfr_cancel:
+    type: event
+    description: >
+      A user pressed the "No thanks" button when the contextual feature recommender appeared.
+    bugs:
+      - 4658
+    data_reviews:
+      - https://github.com/mozilla-mobile/fenix/pull/5194
+    notification_emails:
+      - fenix-core@mozilla.com
+    expires: "2020-03-01"
+  pinned_shortcut_priv:
```

Comment:
would `opened_from_shortcut` make more sense here?

Also by home screen in the description do we mean the Android home screen or the Fenix home screen? If it's the former maybe we should call that out?

Reply:
There's different kinds of shortcuts, so I think `opened_from_shortcut` is also kind of ambiguous. 

That's a good point about the home screen; I'll clarify it in the description.

Reply:
@jyeontaek Are there multiple types of shortcuts in private_browsing_mode? Because this is nested it will get it's context from its parent

Reply:
Yes there are.